wiki/markdown/overriding.md

```diff
@@ -0,0 +1,58 @@
+## Overriding Message Service Library
+
+When building ``eiffel-remrem-generate`` and ``eiffel-remrem-publish`` services 
+from open-source, target packages include also ``eiffel-remrem-semantics`` library 
```

Comment:
Since this documentation is in open-source and everyone read this from a open-source perspective, i think open-source don't need to be mentioned.
Write documentation from a general perspective and describe just how to override the built-in pre-built eiffel-semantics version with a provided eiffel-semantics jar file. And also describe where eiffel-semantics jar file can be downloaded from JitPack/Github.

Reply:
Thanks for these points, Tobias. Fixed.